Assets/Scripts/Interactables/Interactable.cs

```diff
@@ -2,22 +2,13 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class Interactable : MonoBehaviour
+public abstract class Interactable : MonoBehaviour
 {
     public bool canInteract = true;
 
-    public virtual void Focus(PlayerStateController player)
```

Comment:
I dont think Focus, Unfocus or Interact should be abstract, as it requires implementation of all the functions to implement interactable. Many types of objects will only use some of them, as interact is the main purpose.

Reply:
Yeah, that's true 😅 What if I leave `Interact` abstract, and change `Focus` and `Unfocus` back to virtual with empty function bodies? 🙂

Reply:
Yup, way better